frontend/app/components/transportation/tdf/modal-splits.js

```diff
@@ -80,13 +80,13 @@ export default class TransportationTdfModalSplitsComponent extends Component {
 
   @action
   addActiveMode(event) {
-    this.analysis.modesForAnalysis.push(event.value);
+    this.analysis.get('modesForAnalysis').push(event.value);
```

Comment:
Great! Why do we need to change from native getter to the method getter? I'd like to avoid it because TypeScript can't understand `get`.

Reply:
I originally added this because it was throwing an error before, but I realized that the error was caused by not using `await` when I grabbed `transportationAnalysis` model in the test, will fix!

Reply:
Awesome, thank you! :)